src/bindings/python/src/pyopenvino/graph/op.cpp

```diff
@@ -28,11 +28,15 @@ class PyOp : public ov::op::Op {
         PYBIND11_OVERRIDE(void, ov::op::Op, validate_and_infer_types);
     }
 
-    bool visit_attributes(ov::AttributeVisitor& visitor) override {
-        // PYBIND11_OVERRIDE_PURE(bool, Op, visit_attributes, visitor);
-        //  Requires binding for visitor
-        //  Now works only for operations without attributes
-        return true;
+    bool visit_attributes(ov::AttributeVisitor& value) override {
+        pybind11::gil_scoped_acquire gil;  // Acquire the GIL while in this scope.
+        // Try to look up the overridden method on the Python side.
+        pybind11::function override = pybind11::get_override(this, "visit_attributes");
+        if (override) {                           // method is found
+            py::bool_ result = override(&value);  // Call the Python function.
+            return result;
+        }
+        return false;
```

Comment:
```suggestion
        if (auto attr_override = pybind11::get_override(this, "visit_attributes")) {                           // method is found
            return override(&value);  // Call the Python function.
        } else {
            return false;
        }
```
The `override(&value);` returns `py::bool_` can it be directly returned as `bool` or `static_cast` is required to make conversion.

Maybe not use `override` for variables as it is also keyword in C++.

Reply:
updated